src/framework/preferences/css/stylus/PrefsEditor.styl

```diff
@@ -257,13 +256,7 @@
     .fl-icon-right:before {
         content: "\e001";
     }
-    .fl-icon-left:before {
-        content: "\e002";
-    }
-    .fl-icon-contrast:before {
-        content: "\e005";
-    }
-    .fl-icon-line-space:before {
+    .ffl-icon-line-space:before {
```

Comment:
I don't think you meant to delete these styles, and there is an extra "f" added as well.